### PR TITLE
Entrypoint: re-try database connection on error for <=5m

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

**Comment by Al2Klimov** [comment]
```
...
[Mon Jul 18 13:29:44.000000 2022] [docker_entrypoint:error] [pid 11] DOCKERE: Can't connect to database: SQLSTATE[HY000] [2002] Connection refused
[Mon Jul 18 13:29:45.000000 2022] [docker_entrypoint:error] [pid 11] DOCKERE: Can't connect to database: SQLSTATE[HY000] [2002] Connection refused
[Mon Jul 18 13:29:46.000000 2022] [docker_entrypoint:error] [pid 11] DOCKERE: Can't connect to database: SQLSTATE[HY000] [2002] Connection refused
ERROR: PDOException in /usr/share/icingaweb2/modules/setup/library/Setup/Utils/DbTool.php:352 with message: SQLSTATE[HY000] [2002] Connection refused
[Mon Jul 18 13:29:47.215484969 2022] [docker_entrypoint:crit] [pid 1] DOCKERE: exit status 1
```

```
ERROR: PDOException in /usr/share/icingaweb2/modules/setup/library/Setup/Utils/DbTool.php:352 with message: SQLSTATE[HY000] [1045] Access denied for user 'icingaweb'@'172.17.0.3' (using password: YES)
[Mon Jul 18 13:31:48.051163193 2022] [docker_entrypoint:crit] [pid 1] DOCKERE: exit status 1
```

**Comment by Al2Klimov** [comment]
Please consider this for (i.e. before) the upcoming release.

**Comment by Al2Klimov** [comment]
_PING_

File: entrypoint/db-init/application/clicommands/DbCommand.php [hunk 1]
```diff
@@ -94,6 +95,20 @@ protected function getDb()
...
+            try {
+                (new DbTool($config))->checkConnectivity();
+                break;
+            } catch (PDOException $e) {
+                fprintf(
+                    STDERR, "[%s] [docker_entrypoint:error] [pid %d] DOCKERE: Can't connect to database: %s\n",
+                    date('D M j H:i:s.u Y'), getmypid(), $e->getMessage()
+                );
+
+                sleep(1);
```
Comment (lippserd) [inline entrypoint/db-init/application/clicommands/DbCommand.php]: I would only sleep every 3 seconds. Otherwise this pollutes the log too much.

**Review by lippserd — CHANGES_REQUESTED** [review]

File: entrypoint/db-init/application/clicommands/DbCommand.php (hunk 1 above)
Comment (lippserd) [reply entrypoint/db-init/application/clicommands/DbCommand.php]: Or even longer.

**Review by lippserd — APPROVED** [review]

**Comment by Al2Klimov** [comment]
Feel free to merge it if OK.

**Pull request merged**